pkg/rtp/codecs/g722_packet.go

```diff
@@ -0,0 +1,18 @@
+package codecs
+
+// G722Payloader payloads G722 packets
+type G722Payloader struct{}
+
+// Payload fragments an G722 packet across one or more byte arrays
+func (p *G722Payloader) Payload(mtu int, payload []byte) [][]byte {
```

Comment:
Can you add a test asserting that MTU is properly handled? After that is done feel free to merge, thanks for adding this :)